frontend/src/api/schema.ts

```diff
@@ -35,3 +35,7 @@ export type CartItemSchema = {
 	product: ProductSchema;
 };
 
+export type MembershipInfo = {
```

Comment:
since this doesn't relate to the db (it seems to be a way to better organize the static data in the 'presentation card'), move this out of schema and add it directly in the file it's used in.